.travis.yml

```diff
@@ -1,16 +1,42 @@
 language: dart
-sudo: false
+sudo: required
+
+# We require PhantomJS 2.x, which is only available on Trusty.
+dist: trusty
 
 dart:
  - dev
  - stable
  - 1.23.0
  - 1.22.1
 
+addons:
+  apt:
+    packages:
+    - ttf-kochi-mincho
+    - ttf-kochi-gothic
+    - ttf-dejavu
+    - ttf-indic-fonts
+    - fonts-tlwg-garuda
+
+before_install:
+  # TODO: remove this and use "sudo: false" when travis-ci/travis-ci#4714 is
+  # fixed.
+  - sudo apt-get update -yq
+  - sudo sh -c "echo ttf-mscorefonts-installer msttcorefonts/accepted-mscorefonts-eula select true | debconf-set-selections"
```

Comment:
What browser needs this? I thought for Dartium to work we simply needed mincho and gothic on top of a trusty install?